spritzle/alert.py

```diff
@@ -70,6 +88,10 @@ def __init__(self):
         log.debug('Alert stopped.')
 
     def register_handler(self, alert_type, handler):
+        if alert_type not in self.alert_types and alert_type not in self.categories:
```

Comment:
Could you add some tests for these cases?

Reply:
Yep, can do.

Reply:
Added a test in there now.